### DISCUSSION TITLE
Can't determine where phantom subdomain is coming from..

### DISCUSSION THREAD

**Discussion by leeenglestone** [community/community#128858] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a website that I have hosted on GitHub pages called start-coding.com. (Which I have just turned off for the moment..)
The other day I noticed someone had created a random subdomain on it.. (someone tried verifying it through Google Webmaster tools)

https://webdisk.start-coding.com/

This isn't a subdomain in GitHub and isn't setup in my CNAME file (which hasn't changed for 3 years).
https://github.com/leeenglestone/startcoding/blob/main/docs/CNAME

I can't find any reference to it in the GitHub codebase and the domain registrar/nameserver (123-reg) is claiming that GitHub need to remove the subdomain??

Both domains point to GitHub..
https://www.domaindiagnosis.com/index.php?query=webdisk.start-coding.com
https://www.domaindiagnosis.com/index.php?query=start-coding.com

This is very odd. Where is this site coming from? and how has someone been able to setup a subdomain on my GitHub page?

-- Lee

**Comment by yoannchaudet** [comment]
Hello,

It looks to me you have configured start-coding.com to be a wildcard DNS record pointing at GitHub Pages. That means anyone can claim any subdomain on your domain. Change your DNS configuration to use A/AAAA or CNAME records like we document here: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.

While at it, I also recommend you to [verify your domain ownership](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages). This will not prevent someone from "stealing" your domain if you configures a wildcard DNS record (yet), but as soon as you fix your DNS configuration this will prevent anyone other than you from even trying to use your domain on GitHub Pages.

**Reply by leeenglestone** [reply]
Thanks - this did the trick! Had a pesky * in there instead of an @
